tests/robot-cases/Group0-BAT/API_DB.robot

```diff
@@ -18,4 +18,7 @@ ${SERVER_API_ENDPOINT}  ${SERVER_URL}/api
 
 *** Test Cases ***
 Test Case - Add Private Project Member and Check User Can See It
```

Comment:
Test Case - Add Private Project Member and Check User Can See It
    Harbor API Test  ./tests/apitests/python/test_add_member_to_private_project.py 	    

Test Case - ****
    Harbor API Test  ./tests/apitests/python/test_del_repo.py

Test Case - ****
    Harbor API Test  ./tests/apitests/python/test_add_sys_label_to_tag.py